ui/src/app/components/modals.tsx

```diff
@@ -1,57 +1,48 @@
 import * as React from 'react';
-export const styles = {
-  modalMain: {
-    // tricky little bit required to get certain CSSProperties
-    // to work properly in this setup
-    position: 'fixed' as 'fixed',
-    background: 'white',
-    borderRadius: '8px',
-    width: '30%',
-    height: 'auto',
-    top: '50%',
-    left: '50%',
-    transform: 'translate(-50%,-50%)',
-    zIndex: 1050
+import ReactModal from 'react-modal';
+import {ReactComponent} from '../utils/index';
+
+const styles = {
+  modal: {
```

Comment:
I'm working on the rename notebook modal so I patched this locally. I noticed the positioning is top/middle with this PR vs middle/middle for the existing modal. Adding this to the modal style appears to do the trick:

```
    top: '50%',
    transform: 'translateY(-50%)'
```

Reply:
That was a deliberate change which I forgot to call out, sorry. We found in Terra that having it centered meant that the whole modal would shift if the content changed, which was a jarring UX. Having it top-aligned still allows it to expand downwards if needed, but the window doesn't jump around.

If you prefer it centered though, I can switch it.

Reply:
I see the issue. That said - I'm not sure we have this kind of scenario in AoU right now, our modals are generally pretty simple. I'm fine with the change if @lstrano-broad is suggesting we make this style rework across all AoU modals - otherwise I would leave the styling as-is since our mocks are still showing centered modals.

For example, this looks weird to me: ![image](https://user-images.githubusercontent.com/822298/50853153-582cbf00-1336-11e9-8c93-d5b9699a4e70.png)

Reply:
Sounds good, let's center it so things stay consistent during the refactor, and we can always swap the implementation later if we decide that's preferable. Thanks!